adafruit_ht16k33/segments.py

```diff
@@ -20,13 +20,14 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""Segment Displays
-****************
-
-.. currentmodule:: adafruit_ht16k33.segments"""
+"""
+Segment Displays
+=================
+"""
 
 from adafruit_ht16k33.ht16k33 import HT16K33
 
+# pylint disable=bad-whitespace
```

Comment:
I believe the white space issue here is that there should be two spaces before the `#`. If this is the case, I would rather we add the extra space than disable `pylint`. If that's not the issue, let me know and we can discuss the options.

Reply:
Actually, it was a guess. Per the "my Travis is borked" commit, Travis was taking time in resyncing the deleted->re-forked repo. It showed me the overall status, but wouldn't load the log so I didn't know what was failing. I can easily remove the disable and see if it passes.